pkg/kn/commands/config/channel/add.go

```diff
@@ -0,0 +1,33 @@
+package channel
```

Comment:
Newly added files should have license headers.

Reply:
DONE! :+1: 
Thanks.